### ISSUE TITLE
Reorganize level select to group like challenge together

### ISSUE THREAD

**Issue by phet-steele** [phetsims/fractions-common#54]
An age old slack conversation with myself and @amanda-phet, with edits for readability:

Steele Dalton: Okay I'm not a fan of the way this game screen is divided, with levels 1-5 for both versions in one half, and level 6-10 for both in the other half. All that instead of 1-10 for one version in one half. Please explain.

Amanda McGarry: That’s how the original one does it.

Steele Dalton: The original one doesn't have to be the best.

Amanda McGarry: I agree, but are you suggesting instead have 10 levels per screen and split up by type of challenge?

Steele Dalton: Yup. Then you can add a little icon next to the switching arrows like an AB switch

Amanda McGarry: Ohh, that’s a neat idea. Instead of  the carousel swiping thing. Why don’t you like it as it is? Seems clear to me.

Steele Dalton: I don't like it as is because you are interrupting progression. Once you hit level 5 of one type, you are much more likely to want to go to level 6 (hence why your "next" button does exactly that) of the same type than to level 1 of a different type. Furthermore, if you DO complete all ten levels of a challenge, the good feeling of a full screen of stars is lost since it is split between two scenes.

Amanda McGarry: Yeah that makes a lot of sense to me.

[End]

@ariel-phet, what say you? The games in question are for the FMN, FI, and BAF sims.

For phetsims/QA/issues/250.

**Comment by KatieWoe** [comment]
I would agree, and further suggest that it might be a good idea to think about including titles as discussed in https://github.com/phetsims/fractions-common/issues/53.  I.E. Build a Fraction: Numbers or Build a Fraction: Shapes.  This may be too verbose though.

**Comment by KatieWoe** [comment]
Note: this also applies to https://github.com/phetsims/QA/issues/252 and https://github.com/phetsims/QA/issues/253
